CHANGELOG.md

```diff
@@ -6,7 +6,8 @@
 
 ### Breaking Changes
 
-_None_
+* This is a test.  
+  [#1](https://github.com/SwiftGen/SwiftGen/issues/2)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGenPlugin/blob/699ecebe030be49ba645749d9ea2da98fba7edc1/CHANGELOG.md#L10"></span>Link text is #1 but links points to SwiftGen/SwiftGen#2.</td>
    </tr>
  </tbody>
</table>